db/getItemsOnList.go

```diff
@@ -0,0 +1,39 @@
+package db
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/mount-joy/thelist-lambda/data"
+)
+
+func (d *db) GetItemsOnList(listID *string) (*[]data.Item, error) {
```

Comment:
A whole file for a single function feels like we will end up with hundreds of files

How would you feel about us putting all dynamo db call functions in a single file?

Reply:
I'm not convinced by this, wouldn't lots of ~40 line files be better than one ~320 line one (assuming about 8 endpoints)?